common/app/navigation/NavLinks.scala

```diff
@@ -61,6 +61,7 @@ private object NavLinks {
   val careers = NavLink("Careers", "/money/work-and-careers")
   val obituaries = NavLink("Obituaries", "/tone/obituaries")
   val greenLight = NavLink("Green light", "/environment/series/green-light")
```

Comment:
Should we remove this too?

Reply:
I think `greenLight` is referenced in `usEnvironment`? https://github.com/guardian/frontend/blob/main/common/app/navigation/NavLinks.scala#L81

Reply:
Should we be removing it from there? 

It's also referenced [here](https://github.com/guardian/frontend/blob/451f9384055b4d7d7161f7547f40926b1d67e5d1/common/app/navigation/NavLinks.scala#L298)

I think I am bikeshedding, just wondering if there's tidy up we should be doing.